app/src/test/java/com/futurice/freesound/utils/TextUtilsTest.java

```diff
@@ -24,17 +24,17 @@
 
     @Test
     public void isNotNullOrEmpty_returnFalse_whenEmptyString() {
```

Comment:
Test names don't match the function.

Add a separate function for `isNotEmpty` and test that independently.